cotel/src/main/scala/day3/SpiralMatrix.scala

```diff
@@ -0,0 +1,147 @@
+package day3
+
+object SpiralMatrix {
+
+  type Matrix[A] = Array[Array[A]]
+
+  def Matrix(xs: Array[Int]*) = Array(xs: _*)
+
+  def distanceToCenterOfMatrix(number: Int): Int = {
+    val spiralMatrix = generateSpiralMatrix(number)
+
+    val numberPosition = get2DPosition(number, spiralMatrix).get
+
+    val middleRow = spiralMatrix.length / 2
```

Comment:
double or int return?

Reply:
Both numbers in the division are Int so the result is Int too